cdm-test/src/test/java/ucar/nc2/util/net/TestHttpRda.java

```diff
@@ -77,6 +77,26 @@
     result.add(new Object[]{"ds628.1_anl_land"});
     result.add(new Object[]{"ds628.1_anl_isentrop"});
 
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
+    result.add(new Object[]{"ds628.1_anl_isentrop"});
```

Comment:
A comment as to why we're adding a bazillion duplicated entries would be really useful. And why not a loop?